docs/make_work.jl

```diff
@@ -166,12 +166,30 @@ function doit(
         end
         src = normpath(root, file)
         dst = normpath(dstbase, relpath(root, srcbase), file)
-        cp(src, dst; force=true)
+        if endswith(file, ".md")
+          symlink(src, dst)
+        else
+          cp(src, dst; force=true)
+        end
         chmod(dst, 0o644)
       end
     end
   end
 
+  function get_rev(uuid::Base.UUID)
+    deps = Documenter.Pkg.dependencies()
+    @assert haskey(deps, uuid)
+    if !isnothing(deps[uuid].git_revision)
+      return deps[uuid].git_revision
+    else
+      return "v$(deps[uuid].version)"
+    end
+  end
+  aarev = get_rev(Base.UUID("c3fe647b-3220-5bb0-a1ea-a7954cac585d"))
```

Comment:
I guess it's a matter of taste, but instead of using the UUID here, one could also query it:
```suggestion
  aarev = get_rev(Base.PkgId(AbstractAlgebra).uuid)
```
or maybe this is needed
```suggestion
  aarev = get_rev(Base.PkgId(Oscar.AbstractAlgebra).uuid)
```

Reply:
I didn't know this worked. Will change this in a few